src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Controllers/HomeController.cs

```diff
@@ -21,13 +21,6 @@ public IActionResult Index()
             return View();
         }
 
-        public IActionResult About()
```

Comment:
Maybe I missed this somewhere, but why is the About action being removed?

Reply:
One of the things that was decided when we picked our items for 2.2 was paring down the number and size of the pages in our templates. The About page/action was a prime target for this since it doesn't demonstrate anything that the Index and Privacy actions don't.

Reply:
Shouldn't the links also be removed from _Layout.cshtml? Am I missing something here?